client/components/reader-full-post/index.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal Dependencies
+ */
+import Main from 'components/main';
+import StickyPanel from 'components/sticky-panel';
+import Gridicon from 'components/gridicon';
+
+export default class FullPostContainer extends React.Component {
+
+	componentWillMount() {
+		document.getElementsByClassName( 'layout' )[ 0 ].classList.add( 'has-no-sidebar' );
```

Comment:
Like this: https://github.com/Automattic/wp-calypso/blob/master/client/wordpress-com.js#L130